### DISCUSSION TITLE
Signature Verification of Enterprise Server 3.11.5 is not working

### DISCUSSION THREAD

**Discussion by manpreet231181** [community/community#109289] [Enterprise]
### Select Topic Area

Question

### Body

we are trying to update Enterprise Server from 3.10.4 to 3.11.5. But Signature verification is not working
![image](https://github.com/community/community/assets/110519149/0f0270f0-3289-460c-9ed5-6655177680a9)

But Signature verification of hot patch 3.10.7 works well
![image](https://github.com/community/community/assets/110519149/6db2a7a2-158c-4146-b1df-c2721c06dbf5)
 Kindly suggest the reason

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @manpreet231181 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by manpreet231181** [reply]
Thanks @queenofcorgis
